x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/CrossClustersQueryIT.java

```diff
@@ -728,6 +729,17 @@ public void testWarnings() throws Exception {
         assertTrue(latch.await(30, TimeUnit.SECONDS));
     }
 
+    public void testSearchesWhereClusterExpressionCannotBeResolved() {
+        setupTwoClusters();
+
+        VerificationException ex = assertThrows(
+            VerificationException.class,
+            () -> runQuery("FROM xremote*:web_traffic | WHERE verb==\"GET\" | KEEP is_https, user_Agent, verb | STATS count(*)", false)
```

Comment:
nit: can you change the cluster to be "nosuchcluster*", just so it's instantly clear to code readers that this is not a cluster than exists.